Templates/Per Tenant Extension/.github/workflows/CICD.yaml

```diff
@@ -170,7 +175,7 @@ jobs:
         with:
           shell: powershell
           parentTelemetryScopeJson: ${{ needs.Initialization.outputs.telemetryScopeJson }}
-          templateUrl: ${{ env.templateUrl }}
+          templateUrl: ${{ fromJson(steps.ReadSettings.outputs.SettingsJson).templateUrl }}
```

Comment:
Instead of doing that, maybe potentially allow certain settings in GITHUB_ENV?

Reply:
Good enough for now. Potential restructuring might utulize a prettier solutions.